README.md

```diff
@@ -1,2 +1,21 @@
 # hw01
 Repository for homework 1
+# Short Bio on Tarryn Rennie 
```

Comment:
Evaluation HW-01 for Tarryn Rennie
----------------------------------------------------
| Overall       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x      |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |           |              |     x       |
| **Ease of access**          |     x      |              |            |

Remarks:

-Your bio was very interesting to read and refreshing to not have just the typical name, major, hometown information!
-I also thought the document looked very nice, and you used the required 4 markdown elements. 
-I marked an needs work x in the "achievement" section just because I couldn't find that you commented describing your process for doing the assignment (especially with GitHub - forking, cloning, etc.) as the instructions asked for. I was a little confused by this also at first but for future reference you can do this by typing in a comment when you first press the button on GitHub to initiate a pull request. 
-Looking at your commits it looks like you had trouble adding in an image (I struggled with this for like 5 commits as well). I finally was able to figure it out by doing this:
1. uploading the file to Github or finding the photo online 
2. Getting the url from Github (by clicking on picture and copying from your url bar) or from other website
3. Using this exact syntax in Markdown: ! [Caption goes here.] (https.urlgoeshere.com/picture.jpg) making sure that this line of code is on a new line and take out the spaces between the sections (between ! and [ and ] and ( -- I just added those so code would show up without running)

Overall great work! Congrats on finishing your first coding assignment!